### PR TITLE
Drop singleton 

### PR DESCRIPTION
**Pull request by peter-gribanov**
Fix for [Don't write to global functions](https://github.com/jupeter/clean-code-php#dont-write-to-global-functions).

Singleton is a [anti-pattern](https://en.wikipedia.org/wiki/Singleton_pattern).
You change a **global** `config()` function to a **global** instance of `Configuration` class.
This is very, very bad.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -512,35 +512,64 @@ Polluting globals is a bad practice in very languages because you could clash wi
...
-            self::$instance = new Configuration();
-        }
-        return self::$instance;
+// config.php
+return [
+    'foo' => 'bar',
+];
+```
+
+Or a [YAML](https://en.wikipedia.org/wiki/YAML) configuration file
```
Comment (TomasVotruba) [inline README.md]: I'd keep it simple with 1 example to do things. That would keep point more clear.

**Comment by TomasVotruba** [comment]
I like the singleton-drop change.

**Comment by piotrplenik** [comment]
Yes - I agree to drop singleton, but with one example only.

**Comment by peter-gribanov** [comment]
@jupeter i can restory singleton examle and mark it as bad.

**Comment by peter-gribanov** [comment]
I add bad Singleton example and remove YAML config.

**Comment by TomasVotruba** [comment]
I think this is ready to merge

**Comment by TomasVotruba** [comment]
@peter-gribanov Could you decouple the singleton example to standalone item?

**Comment by peter-gribanov** [comment]
@TomasVotruba I create a new section. You can complete the description.

**Comment by TomasVotruba** [comment]
Thank you, this is good enough

**Pull request merged**